.github/scripts/.helm-tests/default/result.yaml

```
@@ -6796,6 +7507,66 @@ spec:
     storage: true
     subresources:
       status: {}
+  - name: v1beta1
+    schema:
+      openAPIV3Schema:
+        description: KeptnMetricsProvider is the Schema for the keptnmetricsproviders
+          API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: KeptnMetricsProviderSpec defines the desired state of KeptnMetricsProvider
+            properties:
+              secretKeyRef:
+                description: SecretKeyRef defines an optional secret for access credentials
+                  to the metrics provider.
+                properties:
+                  key:
+                    description: The key of the secret to select from.  Must be a
+                      valid secret key.
```

```suggestion
                    description: Key of the secret.
                    Must be a
                      valid secret key.
```
Ending the sentence with "to select from" is bad grammar but I don't know what the intent was.

---

this is k8s official documentation `corev1.SecretKeySelector`